src/Smidge/Controllers/SmidgeController.cs

```diff
@@ -192,7 +192,7 @@ public class SmidgeController : Controller
 
                 using (var resultStream = await GetCombinedStreamAsync(files, bundleContext))
                 {
-                    var compressedStream = await Compressor.CompressAsync(file.Compression, resultStream);
+                    var compressedStream = await Compressor.CompressAsync(file.Compression, CompressionLevel.Optimal, resultStream);
```

Comment:
This is a functional change, is this fixing anotehr issue?

Reply:
See below.